src/main/java/org/jenkinsci/maven/plugins/hpi/RunMojo.java

```diff
@@ -183,7 +185,10 @@ public class RunMojo extends AbstractJettyMojo {
     /**
      * If true, the context will be restarted after a line feed on
      * the input console. Enabled by default.
+     *
+     * @deprecated use {@link JettyRunWarMojo#scan}
      */
+    @Deprecated
     @Parameter(property = "jetty.consoleForceReload", defaultValue = "true")
     protected boolean consoleForceReload;
```

Comment:
The superclass now has a more fine-grained `scan` parameter to control reloading via console or by scanning files for changes.